lib/matplotlib/quiver.py

```diff
@@ -626,19 +626,19 @@ def _angles_lengths(self, U, V, eps=1):
 
     def _make_verts(self, U, V, angles):
         uv = (U + V * 1j)
-        str_angles = isinstance(angles, six.string_types)
-        if str_angles and (angles == 'xy' and self.scale_units == 'xy'):
+        str_angles = angles if isinstance(angles, six.string_types) else ''
```

Comment:
I guess not, str_angle is here to avoid the `FutureWarning: elementwise comparison failed` cited above and was introduced in https://github.com/matplotlib/matplotlib/pull/6971

I tried to stick with the existant ...